WordPress/Classes/Models/BloggingPromptSettings+CoreDataClass.swift

```diff
@@ -11,8 +11,8 @@ public class BloggingPromptSettings: NSManagedObject {
         let siteID: NSNumber
         if let id = blog.dotComID {
             siteID = id
-        } else if let account = try WPAccount.lookupDefaultWordPressComAccount(in: context) {
-            siteID = account.primaryBlogID
+        } else if let account = try WPAccount.lookupDefaultWordPressComAccount(in: context), let primaryBlogID = account.primaryBlogID {
+            return nil
```

Comment:
Looks like a function change. Previous:

````swift
siteID = account.primaryBlogID
```

Reply:
Good catch! Fixed in https://github.com/wordpress-mobile/WordPress-iOS/pull/24328/commits/a185c04226b8114bc00a943997675548ce3b35fa